sass/_custom.scss

```diff
@@ -1,15 +1,113 @@
-h1, .title-text {
-  font-size: 1.75rem;
+h1,
+.title-text {
+    font-size: 1.75rem;
+    margin-bottom: 40px;
 }
 
-h2, .subtitle-text {
-  font-size: 1.5rem;
+h2,
+.subtitle-text {
+    font-size: 1.5rem;
 }
 
 h3 {
-  font-size: 1.125rem;
+    font-size: 1.125rem;
 }
 
 .text {
-  font-size: 1rem;
+    font-size: 1rem;
+}
+
+header {
+    .logo {
+        margin-left: 0 !important;
+        font-family: Montserrat !important;
+        font-weight: bold !important;
+    }
+
+    @media screen and (max-width: 768px) {
+        nav {
+            text-align: center;
+        }
+    }
+
+    .nav-item {
+        font-family: Montserrat;
+        font-weight: normal !important;
+
+        &:hover {
+            color: white !important;
+        }
+    }
+}
+
+.hero {
+    padding-top: 55px !important;
+    padding-bottom: 55px !important;
+
+    @media screen and (max-width: 768px) {
+        .welcome {
+            max-width: none !important;
+        }
+    }
+    .welcome {
+        max-width: 50% !important;
+
+        .heading-text {
+            font-family: "Dharma Gothic E";
+            font-size: 5.5rem;
```

Comment:
The text may be too big here. But maybe it is because I do not have the font? Should it have an alternative font-family?

<img width="1168" alt="image" src="https://user-images.githubusercontent.com/6969134/150350087-69113d87-e1de-470a-ad28-37d58de1be08.png">

Reply:
maybe `3.5rem`?